Docs/reference/content/reference/driver/crud/reading.md

```diff
@@ -312,4 +312,224 @@ await OutAsync("myNewCollection");
 ```
 ```json
 { $out: 'myNewCollection' }
-```
\ No newline at end of file
+```
+
+### Text Search
+[Atlas Search](https://docs.atlas.mongodb.com/atlas-search) makes it easy to build fast, relevance-based search capabilities on top of your MongoDB data. Try it today on [MongoDB Atlas](https://www.mongodb.com/cloud/atlas), our fully managed database as a service.
+
+The following examples can be used alongside the [Atlas Search Tutorial](https://docs.atlas.mongodb.com/reference/atlas-search/tutorial/)
+
+```csharp
+var collection = client.GetDatabase("sample_mflix").GetCollection<BsonDocument>("movies");
+var searchStage = @"
+{
+	$search: {
+    	'text': {
```

Comment:
Line has a mixture of spaces and tabs and doesn't line up correctly in Beyond Compare.

We usually avoid tabs altogether and only uses spaces.

Check other lines for tabs also.